h/static/scripts/controllers/form-controller.js

```diff
@@ -11,6 +11,14 @@ function shouldAutosubmit(type) {
 }
 
 /**
+ * Return true if a form field should be hidden until the user starts
+ * editing the form.
+ */
+function isHiddenField(el) {
```

Comment:
Would it be possible to add an @example (http://usejsdoc.org/tags-example.html) here?